fixtures/command_line_tool_with_static_dependencies/Project.swift

```diff
@@ -0,0 +1,18 @@
+import ProjectDescription
+
+let project = Project(name: "CommandLineTool",
+                      targets: [
+                        Target(name: "CommandLineTool",
+                               platform: .macOS,
+                               product: .commandLineTool,
+                               bundleId: "com.example.commandlinetool",
```

Comment:
It's worth adding a few source files with compilable code - e.g. in the command line target importing and using code from the static library.

e.g. I added a `main.swift` and a `MyStaticLibClass.swift`, and that helped me discover we were missing linking static libraries 😅